src/components/03-components/01-actions/actions.config.yml

```diff
@@ -2,10 +2,13 @@ title: Actions Menu
 status: wip
 preview: "@sam"
 context:
-    style: primary
-    label: action
-    icon:
-        icon: ellipsis
-        classes: fa-fw
-        dataTransform: grow-5
+  style: primary
```

Comment:
Was the old config broken due to whitespace? If not, please revert the changes in this file. No need to clutter up this PR and the file history with purely whitespace changes.